### PR TITLE
#trivial - f-searchbox-sass-update

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Changed
- Update to `@use` and `@forward` SASS syntax

### CODE REVIEW

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/molecules/f-searchbox/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v4.0.0-beta.42
+------------------------------
+*Jun 24, 2022*
```
Comment (xander-marjoram) [inline packages/components/molecules/f-searchbox/CHANGELOG.md]: Nit: Not sure where all of these `Jun`s have come from all of a sudden!

File: packages/components/molecules/f-searchbox/package.json [hunk 2]
```diff
@@ -68,10 +68,10 @@
     "vuex": "3.5.1"
   },
   "devDependencies": {
-    "@justeat/f-button": "3.x",
-    "@justeat/f-error-message": "1.x",
+    "@justeat/f-button": "3.3.0",
+    "@justeat/f-error-message": "1.0.0",
```
Comment (xander-marjoram) [inline packages/components/molecules/f-searchbox/package.json]: Was there a particular reason we didn't go for the latest minor version? I understand avoiding the major version bump but v1.2.1 is available.

Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/package.json]: Fixed. I must have missed this during the merge conflict from Ben's PR.

File: packages/components/molecules/f-searchbox/CHANGELOG.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/CHANGELOG.md]: Fixed.

**Review by jamieomaguire — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**